deeplearning4j-core/src/main/java/org/deeplearning4j/nn/layers/feedforward/rbm/RBM.java

```diff
@@ -316,11 +316,11 @@ public Layer transpose() {
      * @param v the visible layer
      * @return the approximated activations of the visible layer
      */
-    public INDArray propUp(INDArray v,boolean training) {
+    public INDArray propUp(INDArray v, boolean training) {
         INDArray W = getParam(PretrainParamInitializer.WEIGHT_KEY);
         if(training) {
-            if(conf.isUseDropConnect() && training) {
-                if (conf.getLayer().getDropOut() > 0 && training) {
```

Comment:
@nyghtowl the check of `training` is redundant and confusing because there is already `if (training)` before that, which means `training` always `true` inside the `if` body.